### PR TITLE
[Feature] Edition: Button to restart drawing geometry

### PR DESCRIPTION
**Pull request by rldhont**
Provide the capability to update geometry with GPS and form coordinates.

* Funded by 3Liz
* Inspired by SDEE 48

### CODE REVIEW

File: lizmap/www/assets/js/edition.js [hunk 1]
```diff
@@ -728,11 +728,13 @@ OpenLayers.Geometry.pointOnSegment = function(point, segment) {
             editLayer.events.on({
 
                 featureadded: function() {
+                    console.log('featureadded');
```
Comment (nboisteault) [inline lizmap/www/assets/js/edition.js:731]: You want to keep `console.log`?

File: lizmap/www/assets/js/edition.js [hunk 2]
```diff
@@ -728,11 +728,13 @@ OpenLayers.Geometry.pointOnSegment = function(point, segment) {
...
                     // Deactivate draw control
                     if( !editCtrls )
                         return false;
                     var geometryType = editionLayer.geometryType;
                     var drawWasActivated = editCtrls[geometryType].active;
+                    console.log('featureadded drawWasActivated: '+drawWasActivated);
```
Comment (nboisteault) [inline lizmap/www/assets/js/edition.js:737]: You want to keep console.log?

**Review by nboisteault — APPROVED** [review]

**Review by mdouchin — CHANGES_REQUESTED** [review]
Shouldn't we add a confirm message to be sure the user wants to launch this tool ? If I undestand correctly, this will delete the current drawing without the possibility to go back (or only by canceling the edition, and so loosing all the modified attribute form data

**Comment by rldhont** [comment]
@mdouchin I have made the requested changes.

File: lizmap/modules/view/locales/en_US/dictionnary.UTF-8.properties [hunk 3]
```diff
@@ -35,6 +35,7 @@ edition.datepicker.open=Open the calendar
 edition.datepicker.reset=Reset the date
 edition.confirm.delete=Are you sure you want to delete the selected feature?
 edition.confirm.cancel=Are you sure you want to cancel edition? You will lose any modifications made on the currently edited object.
+edition.confirm.restart-drawing=Are you sure you want to delete geometry and restart drawing?
```
Comment (mdouchin) [inline lizmap/modules/view/locales/en_US/dictionnary.UTF-8.properties:38]:
```suggestion
edition.confirm.restart-drawing=Are you sure you want to delete the existing geometry and redraw it from scratch ?
```

File: lizmap/modules/view/locales/en_US/edition.UTF-8.properties [hunk 4]
```diff
@@ -36,6 +36,7 @@ geomtool.rotate.title=Rotate
 geomtool.reshape.title=Reshape
 geomtool.splitfeatures.title=Split
 geomtool.reversegeom.title=Reverse the geometry
+geomtool.restartdrawing.title=Restart drawing
```
Comment (mdouchin) [inline lizmap/modules/view/locales/en_US/edition.UTF-8.properties:39]:
```suggestion
geomtool.restartdrawing.title=Redraw the geometry
```

**Review by mdouchin — CHANGES_REQUESTED** [review]

**Comment by rldhont** [comment]
Thanks @mdouchin changes integrated

**Review by mdouchin — APPROVED** [review]

**Pull request merged**